lib/common/login.js

```diff
@@ -35,16 +35,16 @@ function login (clientName, options) {
   options = options || {};
   return auth0ClientFactory.getClientIdByName(clientName).then(clientId => {
     if (options.username) {
-      const loginProvider = new UsernameLoginProvider(clientId, clientName);
-      return loginProvider.login(options.username, options.password);
+      const loginProvider = new UsernameLoginProvider(clientId);
+      return loginProvider.login(options.username === true ? null : options.username, options.password);
```

Comment:
I need an explicit check for true here as i want the api to allow passing true (to start a username/password provider **with** prompt for username) or a username (to start a username/password provider **without** a prompt for username)